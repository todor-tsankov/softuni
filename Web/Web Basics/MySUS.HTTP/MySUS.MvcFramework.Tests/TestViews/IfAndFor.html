<!DOCTYPE html>

<html>
<head>
    <meta charset="utf-8" />
    <title></title>
</head>
<body>
    @if(true)
    {
        <p>Included</p>
    }

    @if(false)
    {
        <p>Not Included</p>
    }

    <ul>
        @for(var a = 0; a < 5; a++)
        {
            <li>1</li>
        }
    </ul>
</body>
</html>