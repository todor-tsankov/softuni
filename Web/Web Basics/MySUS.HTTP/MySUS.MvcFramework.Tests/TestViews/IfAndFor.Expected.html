<!DOCTYPE html>

<html>
<head>
    <meta charset="utf-8" />
    <title></title>
</head>
<body>
        <p>Included</p>


    <ul>
            <li>1</li>
            <li>1</li>
            <li>1</li>
            <li>1</li>
            <li>1</li>
    </ul>
</body>
</html>
